.github/workflows/release.yml

```
@@ -0,0 +1,132 @@
+        with:
+          context: .
+          file: ./php/latest.Dockerfile
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+      - name: Generate artifact attestation
+        uses: actions/attest-build-provenance@v2
+        with:
+          subject-name: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME}}
+          subject-digest: ${{ steps.push.outputs.digest }}
+          push-to-registry: true
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)